node/libs/concurrency/src/net/http/mod.rs

```diff
@@ -0,0 +1,46 @@
+//! Http Server configuration structs
```

Comment:
http has nothing to do with the concurrency crate, please move it to zksync_consensus_network, or to zksync_consensus_utils if that is not feasible